scene/gui/rich_text_label.cpp

```diff
@@ -5480,7 +5480,8 @@ void RichTextLabel::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("add_image", "image", "width", "height", "color", "inline_align", "region"), &RichTextLabel::add_image, DEFVAL(0), DEFVAL(0), DEFVAL(Color(1.0, 1.0, 1.0)), DEFVAL(INLINE_ALIGNMENT_CENTER), DEFVAL(Rect2(0, 0, 0, 0)));
 	ClassDB::bind_method(D_METHOD("newline"), &RichTextLabel::add_newline);
 	ClassDB::bind_method(D_METHOD("remove_paragraph", "paragraph"), &RichTextLabel::remove_paragraph);
-	ClassDB::bind_method(D_METHOD("push_font", "font", "font_size"), &RichTextLabel::push_font);
+	ClassDB::bind_method(D_METHOD("push_font", "font", "font_size"), &RichTextLabel::push_font, DEFVAL(0));
+	ClassDB::bind_compatibility_method(D_METHOD("push_font", "font", "font_size"), &RichTextLabel::push_font);
```

Comment:
```suggestion
#ifndef DISABLE_DEPRECATED
	ClassDB::bind_compatibility_method(D_METHOD("push_font", "font", "font_size"), &RichTextLabel::push_font);
#endif
```
But it is probably fine to keep these in this file (for this PR at least) as they don't require separate methods.
(same for the other)

Reply:
I put these two bind method  in `#ifndef DISABLE_DEPRECATED`, but it can't pass check.
It seem is crash when start up, but work fine on my device.
How should I do?

Reply:
It's a known issue: https://github.com/godotengine/godot/issues/78749